aten/src/ATen/native/cuda/linalg/BatchLinearAlgebraLibBlas.cpp

```diff
@@ -225,7 +225,15 @@ void triangular_solve_batched_cublas(const Tensor& A, const Tensor& B, bool left
 
 template <typename scalar_t>
 inline void apply_gels_batched(const Tensor& A, Tensor& B, Tensor& infos) {
+#if not defined(ROCM_VERSION)
```

Comment:
is that valid syntax?

Reply:
Nice catch.
Not sure how this is handled by the pre processor. I will correct it.